### PR TITLE
python3Packages.beancount_docverif: add build dependency

### PR DESCRIPTION
**Pull request by siriobalmelli**
Missing build dep broke Linux build

Signed-off-by: Sirio Balmelli <sirio@b-ad.ch>

### CODE REVIEW

**Comment by SuperSandro2000** [comment]
@ofborg eval

**Comment by SuperSandro2000** [comment]
@ofborg eval

File: pkgs/development/python-modules/beancount_docverif/default.nix [hunk 1]
```diff
@@ -14,6 +15,10 @@ buildPythonPackage rec {
     sha256 = "1kjc0axrxpvm828lqq5m2ikq0ls8xksbmm7312zw867gdx56x5aj";
   };
 
+  buildInputs = [
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/beancount_docverif/default.nix:18]:
```suggestion
  nativeBuildInputs = [
```

Comment (siriobalmelli) [reply pkgs/development/python-modules/beancount_docverif/default.nix]: Agreed, this is cleaner. Respun and force-pushed.

**Pull request merged**